server/Pipfile

```diff
@@ -6,6 +6,7 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
+alembic = "==1.4.2"
```

Comment:
you shouldn't need to add alembic as it's already required by `flask-migrate` (it's already in the Pipfile.lock)?

Reply:
For some reason, the server didn't want to run unless I explicitly installed Alembic - that's why I added it.